robotd/devices.py

```diff
@@ -100,10 +100,14 @@ def name(cls, node):
         """Simple node name."""
         return node.sys_name
 
+    def read_temperature_value(self):
+        path = '{}/temp'.format(self.node.sys_path)
+        with open(path) as file:
```

Comment:
I don't like the overriding of the `file` keyword here

Reply:
FTR `f` is the standard shorthand shorthand convention here

Reply:
‘file’ no longer exists in python 3

Reply:
Ah I didn't know. never mind then :)